kview/src/main/java/dev/ikm/komet/kview/controls/skin/KLComponentSetControlSkin.java

```diff
@@ -109,7 +109,7 @@ protected void layoutChildren(double contentX, double contentY, double contentWi
         double x = contentX + padding.getLeft();
         double y = contentY + padding.getTop();
         titleLabel.resizeRelocate(x, y, labelPrefWidth, labelPrefHeight);
-        y += labelPrefHeight;
+        y += labelPrefHeight + KLComponentListControlSkin.SPACE_BETWEEN_COMPONENTS;
         double spacing = 10;
         for (KLComponentControl componentControl : getChildren().stream()
                 .filter(KLComponentControl.class::isInstance)
```

Comment:
we need to remove this class entirely. Should have done that sorry. We should also remove the control that uses this skin. We're not using these classes anymore.
Thanks